docs/2/technical/data-format/microshare-standards.md

```diff
@@ -31,18 +31,12 @@ toc: true
 ---------------------------------------
 
 
-As you have seen in data integration, unpacked data is data that is developed using Microshare® unpackers. 
+Data is sent as a payload from the IoT sensors to the Microshare® unpackers as raw, encrypted data for security purposes. The data is then transformed by the Microshare® unpackers into comprehensible information. This information is referred to as unpacked data and is inserted into the device clusters. 
```

Comment:
actually this should say "raw, compressed data for efficiency." and perhaps

"the transformed and enriched by the Microshare unpacker into comprehensible information using one of our canonical json formats. This process ensures that the data can be analyzed consistently regardless of device vendor or the network used to transmit. The final format is referred to as unpacked data."